pipelines/MetricsRuntime.yaml

```diff
@@ -24,54 +24,38 @@ tasks:
       connections.metric: CalibrateTime
       connections.labelName: calibrate
       target: calibrate.run
-  timing_imageDifference:
+  timing_subtractImages:
     class: lsst.verify.tasks.commonMetrics.TimingMetricTask
     config:
-      connections.package: pipe_tasks
+      connections.package: ip_diffim
       connections.metric: ImageDifferenceTime
-      connections.labelName: imageDifference
-      metadataDimensions: [instrument, visit, detector, skymap]
-      target: imageDifference.run
-  timing_imageDifference_astrometer:
-    class: lsst.verify.tasks.commonMetrics.TimingMetricTask
-    config:
-      connections.package: meas_astrom
-      connections.metric: AstrometryTime
-      connections.labelName: imageDifference
-      metadataDimensions: [instrument, visit, detector, skymap]
-      target: imageDifference:astrometer.loadAndMatch
-  timing_imageDifference_register:
-    class: lsst.verify.tasks.commonMetrics.TimingMetricTask
-    config:
-      connections.package: pipe_tasks
-      connections.metric: RegisterImageTime
-      connections.labelName: imageDifference
-      metadataDimensions: [instrument, visit, detector, skymap]
-      target: imageDifference:register.run
-  timing_imageDifference_subtract:
+      connections.labelName: subtractImages
+      metadataDimensions: [instrument, visit, detector]
+      target: subtractImages.run
     class: lsst.verify.tasks.commonMetrics.TimingMetricTask
+  timing_detectAndMeasure:
     config:
       connections.package: ip_diffim
-      connections.metric: ImagePsfMatchTime
-      connections.labelName: imageDifference
-      metadataDimensions: [instrument, visit, detector, skymap]
-      target: imageDifference:subtract.subtractExposures
-  timing_imageDifference_detection:
+      connections.metric: DetectAndMeasureTime
+      connections.labelName: detectAndMeasure
+      metadataDimensions: [instrument, visit, detector]
+      target: detectAndMeasure.run
```

Comment:
It looks like `ip_diffim.DetectAndMeasureTime` is supposed to be a new metric, but I don't see a PR in `verify_metrics`. Even if the pipeline runs, I don't think SQuaSH upload will work without it.

It might be a good chance to get rid of `AstrometryTime` and `RegisterImageTime` as well, since I don't think they were ever used.